testutil/testdata_pulsar/testpb/query.pulsar.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-go-pulsar. DO NOT EDIT.
-package testdata_pulsar
+package testpb
```

Comment:
An alternative we can do, is that instead of generating pulsar types in `testutil/testdata_pulsar/testpb`, we generate them in `testutil/testdata/testpb`. This way testdata_pulsar is not a directory with only one directory inside.

We would have:
- `testdata/*.go` : gogo
- `testdata/testpb/*.go`: pulsar

Not sure if that's better or more confusing, but putting the idea out there.